Source/Scene/Expression.js

```diff
@@ -201,6 +201,24 @@ define([
         //>>includeEnd('debug');
     }
 
+    function parseMemberExpression(expression, ast) {
+        var obj = createRuntimeAst(expression, ast.object);
+        if (ast.computed) {
+            var val = createRuntimeAst(expression, ast.property);
+            return new Node(ExpressionNodeType.MEMBER, 'computed', obj, val);
```

Comment:
In my previous comment, I meant to change "compute" to "bracket" throughout the codebase, not just in the tests.